studiopress/genesis-sample
# Updates/wordpress 53

**Summary of change:**


- Updates widths for nested blocks
- Updates CSS for captions (image, gallery, gallery item)
- Adds new alignment classes
- Adjusts table styles to allow for striped table settings, match editor to front end, update spacing
- Removes styles for squared button style in favor of new border radius setting
- Adjust separator styles to allow for color options
- Updates homepage content import


**Have the changes in this PR been added to the documentation for this project?**


n/a

**How to test:**


1. Verify 3 separator styles display properly in the editor and on the front end with:
     - Default color
     - Custom color
4. Verify captions display properly for images, galleries, and gallery items.
5. Verify alignments work for various items (left, right, center, etc).
6. Verify button styles work, especially in homepage import. Default should have a 5px border-radius.
7. Verify tables display properly at all width options (centered appears to be a bug in WP).
8. Verify table styles match editor to front-end.
9. Verify fixed table option works.
10. Verify striped table options work in editor and front-end.
11. Verify nested blocks work with various combinations.
     - Full inside wide
     - Wide inside full
     - Center inside wide and full
     - Multiple nesting layers
     - etc.
12. Import the homepage content for both Starter Packs. Verify content imports correctly and is editable.


**Suggested Changelog Entry:**

See Changelog changes.
